packages/engine/lib/orchestrator/src/manifest.rs

```diff
@@ -31,6 +31,8 @@ const DATASET_FILE_EXTENSIONS: [&str; 2] = ["csv", "json"];
 /// to parse the project structure easily.
 #[derive(Debug, Default, Clone)]
 pub struct Manifest {
+    /// The name of the project
+    pub name: String,
```

Comment:
`project_name`?

Reply:
For the sake of clarity, will change :)

I think if we wanted to be pedantic then name is probably fine because all of the docs and such about `Manifest` suggest it's a "project's manifest" rather than an experiment's :D